README.md

```diff
@@ -78,7 +78,7 @@ console.log('Current balance:', xpubInfo.current_balance);
 
 The **first argument** of the `SpvWalletClient` constructor is the URL of the SPV Wallet server.
 
-> Note the `/v1` suffix is required.
+> Note the `/api/v1` suffix is required.
```

Comment:
Sorry, but I think that as a first step in adjusting clients to new rest API we should remove requirement for the path `/v1` or `/api/v1` from the spvWalletServerUrl,
In other words, the spv-wallet-x-client should choose where it use v1 or api/v1 or api/v2, etc.

Reply:
I made it part of `client.ts` constructor, please take a look.